mypy/stubtest.py

```diff
@@ -942,6 +942,15 @@ def is_subtype_helper(left: mypy.types.Type, right: mypy.types.Type) -> bool:
     ):
         # Pretend Literal[0, 1] is a subtype of bool to avoid unhelpful errors.
```

Comment:
Btw, I am almost sure that it does not work anymore, because `bool()` now is `Union[Literal[False], Literal[True]]`.

I can address this in a new PR.